### PR TITLE
Added description prop to command.ts and created help command (help.ts)

### PR DESCRIPTION
**Pull request by BronzW**
Hey so the spaces before and after the brackets is just a thing vscode does when I save the file. I also added DEFAULT_PREFIX to the env file if that's okay.

Mainly I added the help command, showing the different commands that are available after the index.ts file. I also added descriptions for each one of the commands and added it as a property to the command class / interface. In the future you would probably also be able to add usages for each command and I hope I've made it easy to do so.

### CODE REVIEW

File: src/cmd/help.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            //command.guildPrefix(msg).then((prefix) => {
+            //    guildPrefix = prefix;
+            //    console.log(prefix);
+            //});
+
+            let commandAliases: string = '';
+            if (command.aliases === undefined) {
+                commandAliases += '``No Aliases``';
+            } else {
+                command.aliases.forEach(function (alias) {
```
Comment (jw2476) [inline src/cmd/help.ts:63]: This should be an arrow function

**Review by jw2476 — CHANGES_REQUESTED** [review]
All looks good, a couple changes which I'll make in a second.  Also, can you DM me the server you are using for testing?

File: src/cmd/help.ts (hunk 1 above)
Comment (jw2476) [reply src/cmd/help.ts]: Fixed

**Review by jw2476 — APPROVED** [review]
All done, merging now!

**Comment by BronzW** [comment]
I'm happy that the pull request went through! I dm:ed you the invite on discord. @jw2476

**Pull request merged**